tests/Mailtrap.IntegrationTests/ContactImports/GetDetails_Success.json

```diff
@@ -0,0 +1,7 @@
+{
+  "id": 1,
+  "status": "started",
```

Comment:
And this one goes for Create success response (BTW the file should be named `GetDetails_Success_Response.json` for consistency).
Another thing is 
```
"created_contacts_count": 10,
"updated_contacts_count": 11,
"contacts_over_limit_count": 0
```
properties should be omitted as they don't make sense for a "started" import (they appear in the "finished" state only). This will be a good opportunity to test the nullability of these properties in the response object.